src/layout/msWorkspace/horizontalPanel.js

```diff
@@ -49,12 +49,14 @@ var HorizontalPanel = GObject.registerClass(
             if (Me.msThemeManager.clockHorizontal) {
                 this.createClock();
             }
-            Me.msThemeManager.connect('panel-size-changed', () => {
-                this.tilingIcon.set_icon_size(
-                    Me.msThemeManager.getPanelSizeNotScaled() / 2
-                );
-                this.queue_relayout();
-            });
+        }
+
+        buildIcon(height) {
+            this.iconSize = height;
```

Comment:
This iconSize seem to be never used as iconSize

Reply:
It is just a parameter used to change the icon size when needed. I used the same name used in taskBar.js, if you have a better name, I can rename them.